metadata-ingestion/tests/integration/looker/looker_mces_golden_deleted_stateful.json

```diff
@@ -783,8 +777,8 @@
     }
 },
 {
-    "entityType": "container",
```

Comment:
Why is this changing?

Reply:
I check the deleted and added urn in master golden file, all urns are present there, this diff is because of rearrangement of entities as empty "globalTags" is removed